test/newrelic_mobile_test.dart

```diff
@@ -351,6 +352,20 @@ void main() {
     ]);
   });
 
+  test('test setMaxOfflineStorageSize should be called with megaBytes', () async {
```

Comment:
megabyte is one word. No big deal though considering we spelled it wrong in our lib.

Reply:
will fix it for next release.